src/Components/NavBar/NavBar.tsx

```diff
@@ -32,19 +32,28 @@ import * as authentication from "./Utils/authentication"
 
 import { AnalyticsSchema } from "Artsy"
 import { track, useTracking } from "Artsy/Analytics"
+import { useMedia } from "Utils/Hooks/useMedia"
 
 export const NavBar: React.FC = track({
   flow: AnalyticsSchema.Flow.Header,
   context_module: AnalyticsSchema.ContextModule.Header,
 })(_props => {
-  const { mediator, user } = useContext(SystemContext)
   const { trackEvent } = useTracking()
+  const { mediator, user } = useContext(SystemContext)
   const [showMobileMenu, toggleMobileNav] = useState(false)
+  const isMobile = useMedia(themeProps.mediaQueries.sm)
   const isLoggedIn = Boolean(user)
 
+  // Close mobile menu if dragging window from small size to desktop
+  useEffect(() => {
+    if (!isMobile) {
+      toggleMobileNav(false)
+    }
+  }, [isMobile])
```

Comment:
Adds a useful new hook -- `useMedia` for evaluating when we're at certain breakpoints and addresses issue where when in mobile view with the nav open, and then dragging browser out to desktop, the menu wouldn't close.